data/json/items/armor/suits_protection.json

```diff
@@ -2330,6 +2330,90 @@
       "menu_text": "Adjust tighter"
     }
   },
+  {
+    "id": "linothorax",
+    "type": "ARMOR",
+	"category": "armor",
+    "name": { "str": "linothorax", "str_pl": "linothorakes" },
+    "description": "Ancient Greek armor made from laminated cloth. Lighter and more comfortable than bronze while providing similar protection.",
+    "weight": "5000 g",
```

Comment:
I'd change this to "5 kg" as it's a nice even number but all in all awesome pr.
Also for your pr validation test error change `Category "content"` to something like `Content "Linothorax armor additions"`